Pod/Classes/VOKCoreDataManager.h

```diff
@@ -302,7 +309,7 @@ typedef void(^VOKObjectIDsReturnBlock)(VOKArrayOfManagedObjectIDs *managedObject
  @param objectClass     Specifies the class to instantiate or fetch when importing data.
  @param completion      Fired on the main queue once the changes have been merged. It brings an NSArray of permanent NSManagedObjectIDs matching the objects deserialized from the import array.
  */
-+ (void)importArrayInBackground:(VOKArrayOfObjectDictionaries *)inputArray
+- (void)importArrayInBackground:(VOKArrayOfObjectDictionaries *)inputArray
```

Comment:
I can't find it now, but I thought there was a compiler macro to indicate that some method has been replaced with another one, so that Xcode can suggest the appropriate correction. I can't find it now, but if that rings any bells, it might be helpful to include it here.